src/github/middleware.ts

```diff
@@ -31,7 +27,7 @@ const withSentry = function (callback) {
 		try {
 			await callback(context);
 		} catch (err) {
-			logger.error(err, "Error while processing webhook");
+			context.log.error(err, "Error while processing webhook");
```

Comment:
So this error log turned out to be useless hence it was impossible to connect it with any webhook. 

By using logger from the context we'll also pass Installation ID and webhook ID so we'll have better Idea where it happens